packit_service/worker/helpers/testing_farm.py

```diff
@@ -463,6 +563,33 @@ def get_latest_copr_build(
         except StopIteration:
             return None
 
+    def _get_artifacts(
+        self,
+        chroot: str,
+        build: CoprBuildTargetModel,
+        additional_build: Optional[CoprBuildTargetModel],
+    ) -> List[Dict]:
+        """
+        Get the artifacts list from the build (if the skip_build option is not defined)
+        and additional build (from other PR) if present.
+        """
+        artifacts = []
```

Comment:
~~What happens to predefined artifacts?~~ Never mind, I thought we added them with some of the PRs for Leapp